pages/index.html

```diff
@@ -1,65 +1,74 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" >
-    <meta name="viewport" content="width=device-width, height=device-height" >
-    <title>Title</title>
-    <link rel="stylesheet" href="./css/CoveoFullSearch.css"/>
-    <script src="js/CoveoJsSearch.Lazy.js"></script>
-    <script src="js/templates/templates.js"></script>
-    <script>
-      document.addEventListener('DOMContentLoaded', function () {
-        Coveo.SearchEndpoint.configureSampleEndpointV2();
-        Coveo.init(document.body);
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, height=device-height">
+  <title>Title</title>
+  <link rel="stylesheet" href="./css/CoveoFullSearch.css" />
+  <script src="js/CoveoJsSearch.Lazy.js"></script>
+  <script src="js/templates/templates.js"></script>
+  <script>
+    document.addEventListener('DOMContentLoaded', function () {
+      Coveo.SearchEndpoint.endpoints['default'] = new Coveo.SearchEndpoint({
+        restUri: 'http://localhost:8100/rest/search',
+        searchAlertsUri: 'http://localhost:8088/rest/search/alerts/',
+        accessToken: '52d806a2-0f64-4390-a3f2-e0f41a4a73ec'
```

Comment:
By the way, there's a neat trick to use a local search API instance with search documentation sample. Just load the page with `?useLocal=1`

See SearchEndpoint.ts -> `isUseLocalArgumentPresent()`

So you could remove this.